api.js

```diff
@@ -246,7 +247,7 @@ Api.prototype.mediaGet = function (req, res, m) {
       res.end()
     } else {
       res.setHeader('content-type', 'image/jpeg')
```

Comment:
This should be based on the file extension. I guess it's ok for now, but we definitely need a marker in here to change

Reply:
Done.